examples/tts/notebooks/SqueezeWave_TTS.ipynb

```diff
@@ -0,0 +1,188 @@
+{
```

Comment:
Can you add SqueezeWave to the current TTS tutorial instead of adding a new notebook?